src/main/java/racingcar/domain/car/RacingCars.java

```diff
@@ -0,0 +1,88 @@
+package racingcar.domain.car;
+
+import static racingcar.domain.exception.CarExceptionMessage.DUPLICATE_EXISTS;
+import static racingcar.domain.exception.CarExceptionMessage.OUT_OF_TOTAL_COUNT;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class RacingCars {
+    private static final int MIN_CAR_COUNT = 2;
+    private static final int MAX_CAR_COUNT = 500;
+    private final List<Car> cars;
+
+    public RacingCars(List<Car> cars) {
+        validateDuplicate(cars);
+        validateTotalCount(cars);
+        this.cars = cars;
+    }
+
+    private void validateDuplicate(List<Car> cars) {
+        Set<Car> uniqueCars = new HashSet<>(cars);
+
+        if (cars.size() != uniqueCars.size()) {
+            throw new IllegalArgumentException(DUPLICATE_EXISTS.getMessage());
+        }
+    }
+
+    private void validateTotalCount(List<Car> cars) {
+        if (cars.size() < MIN_CAR_COUNT || cars.size() > MAX_CAR_COUNT) {
+            throw new IllegalArgumentException(OUT_OF_TOTAL_COUNT.getMessage());
+        }
+    }
+
+    public void startRace() {
+        for (Car car : cars) {
+            car.move();
+        }
+    }
+
+    public Map<String, Integer> getCars() {
+        Map<String, Integer> nameAndPosition = convertToNameAndPosition(cars);
+        return Collections.unmodifiableMap(nameAndPosition);
+    }
+
+    private Map<String, Integer> convertToNameAndPosition(List<Car> target) {
+        Map<String, Integer> nameAndPosition = new LinkedHashMap<>();
+        for (Car car : target) {
+            nameAndPosition.put(car.getName().getName(), car.getPosition());
```

Comment:
디미터 법칙을 고려한다면 `Car` 객체의 `getName()` 메서드가 `CarName`이 아닌 `String`을 반환하도록 구현하면 더 좋을 것 같습니다!
```suggestion
            nameAndPosition.put(car.getName(), car.getPosition());
```
위와 같이 작성할 수 있도록이요!